generic-jms-ra-jar/pom.xml

```diff
@@ -50,8 +50,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.1</version>
                 <configuration>
-                    <source>1.6</source>
```

Comment:
changing the JDK version form 1.6 to 1.8 seems important enough to be mentioned in the commit message

Reply:
I agree